pom.xml

```diff
@@ -589,6 +589,30 @@
     </pluginManagement>
 
     <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <configuration>
+          <failOnError>${findbugs.failOnError}</failOnError>
+          <!--  Do not define excludeFilterFile here as it will force a plugin to provide a file -->
```

Comment:
Should this comment be removed since `excludeFilterFile` is defined a few lines below?

Reply:
yeah, likely.